samples/servicecontrol/retry-messages/sample.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Retry failed messages using ServicePulse
+summary: A sample showing how to retry failed messages
+component: Core
+reviewed: 2024-09-03
+---
+
+This sample shows how to retry a failed message. The sample uses the [Learning Transport](/transports/learning/) and a portable version of the Particular Service Platform tools. Installing ServiceControl is **not** required.
+
+include: platformlauncher-windows-required
+
+downloadbutton
+
+## Running the project
+
+Running the project will result in 3 console windows. Wait a moment until the ServicePulse window opens in the browser.
+
+![service-pulse-fresh](service-pulse-fresh.png)
+
+### Sender
+
+The sender is a program that uses NServiceBus to send simple test messages.
+
+![send-a-message](send-a-message.png)
+
+Press <kbd>Enter</kbd> in the Sender console window to send one.
+
+### Receiver
+
+The receiver is a program that uses NServiceBus to read messages off a queue and process them. It has a fault simulation mode that is enabled by default. Because of that, the message that has just been sent fails to be processed.
+
+![receiver-error](receiver-error.png)
```

Comment:
I don't think we should use screenshots from consoles as these are text. Wouldn't it be better to just paste the console text here? Like:

```
Mode                 LastWriteTime         Length Name
----                 -------------         ------ ----
d----          28/08/2024    14:20                coverage
d----          05/09/2024    12:09                node_modules
d----          27/08/2024    14:56                public
d----          27/08/2024    14:56                src
d----          27/08/2024    14:56                test
-a---          27/08/2024    14:56             27 .eslintignore
-a---          27/08/2024    14:56           1071 .eslintrc.cjs
-a---          27/08/2024    14:56            347 .gitignore
-a---          27/08/2024    14:56              8 .prettierignore
-a---          27/08/2024    14:56            257 .prettierrc.json
-a---          27/08/2024    14:56            291 env.d.ts
-a---          27/08/2024    14:56            461 index.html
-a---          28/08/2024    12:39         328391 package-lock.json
-a---          27/08/2024    14:56           2422 package.json
-a---          27/08/2024    14:56            595 tsconfig.app.json
-a---          05/09/2024    12:32         169631 tsconfig.app.tsbuildinfo
-a---          27/08/2024    14:56            149 tsconfig.json
-a---          27/08/2024    14:56            280 tsconfig.node.json
-a---          27/08/2024    14:56           2490 vite.config.ts
-a---          27/08/2024    14:56            878 vitest.config.ts

  ramon  S:    src  Frontend [  trial-license-bar ≡  ~3 ] cd,,cd
ParserError:
Line |
   1 |  cd,,cd
     |    ~
     | Missing argument in parameter list.
  ramon  S:    src  Frontend [  trial-license-bar ≡  ~3 ]
  ramon  S:    src  Frontend [  trial-license-bar ≡  ~3 ] cd..
  ramon  S:  ServicePulse  src [  trial-license-bar ≡  ~3 ] cd..
  ramon  S:  ServicePulse [  trial-license-bar ≡  ~3 ] code .
  ramon  S:  ServicePulse [  trial-license-bar ≡  ~3 ]
```

Better readable, crisper on UHD screens, mobile friendly, etc.